### PR TITLE
(maint) Update module for new Puppet branding

### PR DESCRIPTION
**Pull request by glennsarti**
Updated the module with the new branding and updated the license date
Updated the CONTRIBUTING file and removed dead links

[ci skip]

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2013 Puppet Labs
+   Copyright 2016 Puppet, Inc.
```
Comment (ferventcoder) [inline LICENSE]: Shouldn't this be 2013-2016?

**Comment by ferventcoder** [comment]
Looks good besides my one comment. While you are at it you should add the maintainers section to the readme.

File: LICENSE (hunk 1 above)
Comment (glennsarti) [inline LICENSE]: Hrmm...Don't think so. The license on Puppet doesn't/didn't have it.  Although I was the one that updated it.

**Comment by glennsarti** [comment]
Probably.  Does that come in as part of modsync? if not, should it?

**Comment by jtappa** [comment]
Looks good to me, the date range for copyright isn't necessary. Current year is standard for web. For instance, see below Github only lists current year.

**Comment by glennsarti** [comment]
Looks like I targetted the wrong branch. Closing the PR

**Pull request closed without merging**